node/src/effect.rs

```diff
@@ -1761,6 +1761,14 @@ impl<REv> EffectBuilder<REv> {
         .await
     }
 
+    pub(crate) async fn get_node_state(self) -> NodeState
+    where
+        REv: From<NodeStateRequest> + Send,
+    {
+        self.make_request(NodeStateRequest, QueueKind::Regular)
```

Comment:
Maybe `QueueKind::Api`?

Reply:
Good catch - thanks!

Reply:
Updated in [9d0bb89](https://github.com/casper-network/casper-node/pull/3215/commits/9d0bb89b901709a15b51c20cc534a8345b7c4a47).